src/main/java/frc/robot/commands/Drive/ArcadeDrive.java

```diff
@@ -56,6 +61,18 @@ public void execute() {
       } else if (rotation > 0) {
         rotation = rotation - Constants.XBOX_DEADBAND_LIMIT;
       }
+
+      //Demo Mode
+        double percentSpeed = m_drive.getPercentPower();
+        
+        speed = speed * percentSpeed;
+        rotation = rotation * percentSpeed;
+      
+
+      //Easter Egg
```

Comment:
please remove commented out code. Also, I don't think we need an insane mode. :)